CONTRIBUTING.md

```diff
@@ -210,7 +208,7 @@ If you're on a feature branch (either for a fix or a new minor/major version), y
 npm run release
 ```
 
-You can also release a beta version from `develop`, using the beta flag in the command line:
+You can also release a beta version from `next`, using the beta flag in the command line:
```

Comment:
`next` instead of `develop` makes sense here?

Reply:
`next` makes sense for the next major version, we can add a mention in "Branch organization".

However, I think this sentence doesn't make sense anymore. We should be able to release a beta version from any branch with Ship.js.

Reply:
```suggestion
You can also release a beta version from another branch, using the beta flag in the command line:
```

?